### PR TITLE
[DE] Translation Roundstructure

### PR DESCRIPTION
**Pull request by DerSchlupf**
German Translation section roundstructure:
image

Printable Book:
image

### CODE REVIEW

**Comment by DerSchlupf** [comment]
I can't build this document. If someone knows the reason for that error I'm interested to learn.

File: translations/round_structure.tex/de.po [hunk 1]
```diff
@@ -77,6 +78,27 @@ msgid ""
 "Keep playing new Rounds until any of the Scenario's ending conditions have been met.\n"
 "\n"
 msgstr ""
+"Das Spiel ist in \\pageltarget{Rounds}{Runden} gegliedert, in dem jeder Spieler im Uhrzeigersinn, beginnend mit dem Startspieler, seinen eigenen Spielzug ausführt.\n"
```
Comment (DarthGandalf) [inline translations/round_structure.tex/de.po:81]: typo. Command names should be left as is - pagetarget

File: translations/round_structure.tex/de.po [hunk 2]
```diff
@@ -77,6 +78,27 @@ msgid ""
...
+"Das Spiel ist in \\pagetarget{Rounds}{Runden} gegliedert, in dem jeder Spieler im Uhrzeigersinn, beginnend mit dem Startspieler, seinen eigenen Spielzug ausführt.\n"
+"Während ihrer Züge bewegen die Spieler ihre \\pagelink{Heroes}{Helden} auf dem Spielplan, errichten neue Gebäude in ihrer \\pagelink{Town}{Stadt} und rekrutieren \\pagelink{Units}{Einheiten} in dem Versuch, die Siegbedingungen des Szenarios zu erfüllen.\\par\n"
+"Die folgenden Schritte werden bei \\textbf{Rundenbeginn jeder Runde} außer der ersten durchgeführt:\n"
+"\\begin{itemize}\n"
+"  \\item Drehe alle zuvor verwendeten Gebäude-, Bevölkerungs- und/oder Zauberbuchmarker zurück auf ihre aktive Seite.\n"
+"  \\item Drehe alle zuvor genutzten Bewegungsmarker  zurück auf ihre aktive, grüne Seite.\n"
+"  \\item Stelle Kronen für genutzte Experteneffekte wieder her \\svgeven{expert}.\n"
+"\\end{itemize}\n"
+"Dann erhalten die Spieler je nach aktueller Rundenzahl entweder Ressourcen oder handeln eine \"Astrologen verkünden\"-Karte ab:\n"
+"\\Anfang{itemize}\n"
```
Comment (qwrtln) [inline translations/round_structure.tex/de.po:90]:
```suggestion
"\\begin{itemize}\n"
```
Was this done by AI? 😆

Comment (DerSchlupf) [reply translations/round_structure.tex/de.po:90]: Oh no, there are more mistakes, than i thought. i think it was AI. Regulary i dont use it, but there was a situation where i accidentally pressed a key combination i didn't know and it translated everything :D i thought i had undone that. i'll do it again. :-D

**Comment by DerSchlupf** [comment]
Yeah, i could build it :) 
I think i can short it in the next days. Otherwise: There is much place on the right column. Perhaps it is possible to make the minataur a little smaller.

**Comment by qwrtln** [comment]
> Yeah, i could build it :) I think i can short it in the next days. Otherwise: There is much place on the right column. Perhaps it is possible to make the minataur a little smaller.

No, we don't shrink art for the sake of translations. We maintain the same document structure. Refer to the original, which does the same.

**Comment by qwrtln** [comment]
BTW, you have 7 runts in here. That's equal in length to the whole second column on the first page.

**Comment by DerSchlupf** [comment]
New picture! Now it is on one single page, but some fixes are still required. I've seen some small errors. Next days I have less time, but i will do it.

**Comment by DerSchlupf** [comment]
New Update. Whats the reason for teh failed page-check? I've checked main and printable main de rulebook and both are on one page.

**Comment by qwrtln** [comment]
> New Update. Whats the reason for teh failed page-check? I've checked main and printable main de rulebook and both are on one page.

The page is packed, and the "newpage" command added after every section gets pushed to the next page, rendering page 12 empty. See it in the printable file. I made a suggestion how you can shave off one line. Feel free to use it or devise something of your own.

**Comment by DerSchlupf** [comment]
Okay, its done now! I'm fine with the result. 

> No, we don't shrink art for the sake of translations. We maintain the same document structure. Refer to the original, which does the same.

I fear for the future it could be that we have to discuss to shrink art in rare cases. German is a language with higher word-lenght on average than english and its a little bit frustrating to short text to a minimum and to risk that wording is boring or - in the worst case - even more unclear only for saving some milimeters of a minotaur. i mean we talked about a **Rule**book. But i will see, what goes in future sections!

**Comment by qwrtln** [comment]
> I fear for the future it could be that we have to discuss to shrink art in rare cases. German is a language with higher word-lenght on average than english and its a little bit frustrating to short text to a minimum and to risk that wording is boring or - in the worst case - even more unclear only for saving some milimeters of a minotaur. i mean we talked about a **Rule**book. But i will see, what goes in future sections!

While it's tempting to disregard current layout and art structure by claiming that your language is exceptional, and your translation doesn't need further work, this never proved to be true. This project has been translated to 7 languages, and every other translator made a claim identical to yours. Almost every language this rulebook has been translated to (apart from Hebrew) requires significantly more words to carry the same message as English. And you know what? All the 6 translating crews managed to make it work, make it look good, and make it just as clear as the original. I have no doubt you'll manage the same with German. The point is not to have a literal translation, but to carry the same meaning in the same structure.

**Review by qwrtln — APPROVED** [review]

**Comment by DarthGandalf** [comment]
> every other translator made a claim identical to yours

Maybe because they are right? :)

In an ideal world what you're saying might be true, but in real world you're applying unequal pressure onto languages. For more dense languages (like Hebrew) you don't need much effort to translate word by word and be done with it (I'm exaggerating, ofc), but for languages less dense than English you'll end up making the sentence structure very awkward and less clear if you try to optimize art size too much.

**Comment by DerSchlupf** [comment]
> While it's tempting to disregard current layout and art structure by claiming that your language is exceptional, and your translation doesn't need further work, this never proved to be true. This project has been translated to 7 languages, and every other translator made a claim identical to yours. Almost every language this rulebook has been translated to (apart from Hebrew) requires significantly more words to carry the same message as English. And you know what? All the 6 translating crews managed to make it work, make it look good, and make it just as clear as the original. I have no doubt you'll manage the same with German. The point is not to have a literal translation, but to carry the same meaning in the same structure.

What you said is not really my point. I don't claim that my translation is the best and the words i use are the only ones which explain the rules on the best way. I just don't understand why in a rule book the art images seem to take priority over the rules, and hobby translators like us are forced to invest a lot more working time to maintain the size of an image that could easily be adjusted just by changing a number in the LaTeX code. I don't mean to say that the structure of the entire booklet should be changed lightly. But the community wouldn't have even noticed if the minotaur had been changed. Small example: The official german word for Turn ist "Spielzug". I had to change it into "Zug". Its not a problem for me now, but isnt that nice. And perhaps there exist other translation-possibilities to save "Spielzug", but ist really time-investment (for a hobby translater) to chec all possibilities. 
But don't worry: I try to save the art in the future, too. If its to annoying, i will tell you ;)

**Comment by qwrtln** [comment]
> you're applying unequal pressure onto languages
I had no idea the project requirements are an unusual pressure. Can you give me examples of other projects like this (say, board game manuals) where the comfort of translators takes precedence over the quality and structure of the document?

> you'll end up making the sentence structure very awkward and less clear
Can you share those claims from players? It's better if they're discussed openly rather than saved for situations like this.

**Comment by DarthGandalf** [comment]
> are an unusual pressure

You misread me again :( unequal, not unusual. I'm not comparing this project with other projects. I'm comparing languages. The more dense the language naturally is, the less pressure you're applying to translators of this language by the means of art sizes.

**Comment by qwrtln** [comment]
> I just don't understand why in a rule book the art images seem to take priority over the rules, and hobby translators like us are forced to invest a lot more working time to maintain the size of an image that could easily be adjusted just by changing a number in the LaTeX code

It's all too easy to open this Pandora's box, it will be a lot harder to close it. Where would you draw a line between what's allowed, and what's not allowed? How much can you shrink the art? Or maybe when can you get rid of it altogether? Such an approach would bring a lot more problems than it would solve. Refer to the original - every translation maintains exactly the same structure, and that's exactly how it should be. When you're investing your time into this, think about the good you're bringing to the world by making this project available to a wider audience. Where your comfort ends, your skills start to sharpen. How did others handle it? Usually it was a combination of team effort, reviews, and AI. I don't think any other translation was done single-handedly.

**Comment by DarthGandalf** [comment]
> Can you share those claims from players?

There's an example just above:

> The official german word for Turn is "Spielzug". I had to change it into "Zug".

How come cutting a correct dictionary word by half makes the text any more understandable?

**Comment by qwrtln** [comment]
> You misread me again :( unequal, not unusual. I'm not comparing this project with other projects. I'm comparing languages. The more dense the language naturally is, the less pressure you're applying to translators of this language by the means of art sizes.

Yeah, sorry, but the point stays the same. Do you have any example to support your claim?

**Comment by qwrtln** [comment]
> > Can you share those claims from players?
> 
> There's an example just above:
> 
> > The official german word for Turn is "Spielzug". I had to change it into "Zug".
> 
> How come cutting a correct dictionary word by half makes the text any more understandable?

I would like to know the opinions of players on the existing translations.

**Comment by qwrtln** [comment]
> How come cutting a correct dictionary word by half makes the text any more understandable?

Nobody's forcing anyone to cut anything. Ideally, we would like to have everything reviewed by native speakers.

**Comment by DarthGandalf** [comment]
> Nobody's forcing anyone to cut anything

You are :)

**Comment by qwrtln** [comment]
> You are :)

Where exactly?

**Comment by qwrtln** [comment]
> If its to annoying, i will tell you ;)

If no teamwork/AI or anything will help, please do tell, we'll see what we can do.

**Comment by DarthGandalf** [comment]
> Where exactly?

Here in this PR you're forcing translators to reduce length of translation.

> No, we don't shrink art for the sake of translations.

Same happened in earlier PRs as well.

**Comment by qwrtln** [comment]
> Same happened in earlier PRs as well.

Also in those PRs, I made suggestions how to shorten the text without cutting any words or introducing any incorrectness. What did you do?

**Comment by DarthGandalf** [comment]
> I made suggestions how to shorten the text without cutting any words or introducing any incorrectness.

Unlike you, I cannot say that I know every language. So I would defer it to native speakers wrt how to make the text shorter, and when having it too short actually hurts the meaning

**Comment by qwrtln** [comment]
> Unlike you, I cannot say that I know every language. So I would defer it to native speakers wrt how to make the text shorter, and when having it too short actually hurts the meaning

All right, you win, I grant you champion-of-the-opressed badge. Good to know I know every language. I do speak German, though.

**Pull request merged**